### PR TITLE
Update for cross environment scripts

### PR DESCRIPTION
**Pull request by nicolengsy**
Had issues with my package.json scripts due to my Windows machine, so I talked to Raghav and edited it so that the scripts can be used across different environments.

- used `cross-env` for the `dev` script and removed `start-on-windows` script.
- escaped quotes (changed `'` to `\"`)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,4 +75,4 @@
       "esm"
     ]
   }
-}
+}
```
Comment (lucashu1) [inline package.json:78]: Add a newline at the end here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by nicolengsy** [comment]
@lucashu1 thanks for checking! Just added the `cross-env` to the devDependencies and fixed the new line issue.

**Comment by lucashu1** [comment]
Thanks @nicolengsy, can you push up the updated `package-lock.json` as well? Typically, whenever you install a new package, you'll run `npm i {packageName}` and then push up the updated `package.json` _and_ `package-lock.json`. I should've clarified that.

**Comment by nicolengsy** [comment]
@lucashu1 understood, done!

**Pull request merged**